students/17th/saemsolyoo/js/login.js

```diff
@@ -0,0 +1,11 @@
+const loginForm = document.querySelector(".login-form");
+
+loginForm.addEventListener("input", () => {
+  const inputId = document.querySelector(".input-id");
+  const inputPw = document.querySelector(".input-pw");
+  const loginBtn = document.querySelector(".login-btn");
+
+  inputId.value.length > 0 && inputPw.value.length > 6
+    ? loginBtn.removeAttribute("disabled")
+    : loginBtn.setAttribute("disabled", "disabled");
```

Comment:
삼항연산자도 활용해서 잘 구현해주셨네요! 
아래와 같이 변수를 활용하면 좀 더 깔끔해질 수 있습니다. 

```suggestion
  const idCondition = inputId.value.length > 0;
  const pwCondition = inputPw.value.length > 6;
  idCondition && pwCondition
    ? loginBtn.removeAttribute("disabled")
    : loginBtn.setAttribute("disabled", "disabled");
```